packages/modular-scripts/src/utils/verifyPackageTree.ts

```diff
@@ -33,7 +36,7 @@ export async function verifyPackageTree(): Promise<void> {
   const dependencies: Dependency = ownPackageJson.dependencies || {};
   const expectedVersionsByDep: Record<string, string> = {};
   // Gather wanted deps
-  depsToCheck.forEach((dep) => {
+  DEPS_TO_CHECK.forEach((dep) => {
     const expectedVersion = dependencies[dep];
     if (!expectedVersion) {
       console.log('HMM', dep, expectedVersion);
```

Comment:
`logger.log`?

Reply:
Also, could you make this log a bit clearer on what the issue is? Just "HMM" is a bit passive aggressive lol

Reply:
This is a *very* unhappy case where we forgot to add a dep to `modular-scripts` `package.json` and therefore can't find the version to match. I've removed it for ease though.